features/integration/algod.feature

```diff
@@ -7,6 +7,8 @@ Feature: Algod
     Then the node should be healthy
 
   Scenario: Status check
+    Given a kmd client
```

Comment:
Getting node status requires us to advance blocks by sending transactions in dev mode (need wallet to make payment txns).